components/support/nimbus-fml/src/parser.rs

```diff
@@ -278,6 +343,34 @@ impl Parser {
             feature_defs: self.features.clone(),
         })
     }
+
+    fn merge_defaults(defaults: serde_json::Value) -> Result<serde_json::Value, FMLError> {
+        let mut channel_map = HashMap::new();
+        let defaults = serde_json::from_value::<Vec<Defaults>>(defaults)?;
+        for default in defaults {
+            if channel_map.contains_key(&default.channel) {
```

Comment:
should add a field `targeting` to the `Defaults` struct and add a `&& default.targeting.is_none()` here for now